doc/releases/release-notes-2.2.rst

```diff
@@ -179,7 +179,18 @@ Boards & SoC Support
 
 .. rst-class:: rst-columns
 
-   * <TBD>
+   * Atmel SAM4E
+   * Atmel SAMV71
+   * Broadcom BCM58400
+   * ST STM32F098xx
+   * NXP i.MX RT1011
+   * Silicon Labs EFM32GG11B
```

Comment:
Silicon Labs EFM32JG12B was added in !21183

Reply:
Thanks. I will add this one because the ARM core is actually different so the diff is significant